packages/dd-trace/test/plugins/agent.js

```diff
@@ -275,7 +276,14 @@ function runCallback (callback, options, handlers) {
 }
 
 module.exports = {
-  // Load the plugin on the tracer with an optional config and start a mock agent.
+  /**
+   * Load the plugin on the tracer with an optional config and start a mock agent.
+   *
+   * @param {String|Array<String>} pluginName - Name or list of names of plugins to load
+   * @param {Object} config
+   * @param {Object} tracerConfig
```

Comment:
```suggestion
   * @param {Record<string, unknown>} config
   * @param {Record<string, unknown>} tracerConfig
```

Object has weird results as type